src/test/converter/nested-modules/nested-modules.ts

```diff
@@ -0,0 +1,45 @@
+module TestModule
+{
+    /**
+     * TestClass class comment.
+     */
+    export class TestClass
+    {
+        /**
+         * This is a property.
+         */
+        public property = 'property';
+        
+        /**
+         * This is a method.
+         */
+        public method(): void
+        {
+
+        }
+    }
+
+    /**
+     * TestClass module comment.
```

Comment:
What happens if this comment is different than the one on the class? Could we also add an example of merging properties (between the class and the module) just so it is clear how that is handled?

Reply:
As far as I could see, both comments are joined together (by a line feed) as "shortText" of the comment block.

I did not check how further paragraphs are handeled. I guess they are joined as well.

It's all about "how to handle type and namespace merging" (#472).

But apart from that I think it is not good style to document the namespace here since almost all it is good for is to expose nested types (as nested classes in C# for example, and here you don't have any further element besides the outer class to attach a documentation).